packages/components/test/translate.js

```diff
@@ -0,0 +1 @@
+export default (key, options) => options.defaultValue;
```

Comment:
I would love to see this function as a the default 't' value props on all components;
src/translate.js ?